### PR TITLE
Implementing some api methods

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

**Comment by saneshark** [comment]
@ayaleloehr ready for review

File: lib/va_rx/api.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'va_rx/api/prescriptions'
+require 'var_rx/api/session'
```
Comment (ayaleloehr) [inline lib/va_rx/api.rb]: Is `var_rx` supposed to be `va_rx`?

File: lib/va_rx/api/prescriptions.rb [hunk 2]
```diff
@@ -3,14 +3,26 @@
...
-      def get_active_rx(*args)
-        perform_with_objects(:get, 'prescription/getactiverx', args.pop, Prescription)
+      def get_active_rxs
+        env = perform(:get, 'prescription/getactiverx', nil, token_headers)
+        Collection.new(Prescription, env, 'prescriptionList')
       end
 
-      def get_history_rx(*args)
-        perform_with_objects(:get, 'prescription/gethistoryrx', args.pop, Prescription)
+      def get_history_rxs
```
Comment (ayaleloehr) [inline lib/va_rx/api/prescriptions.rb]: This method name feels off.  Should it maybe be changed to `get_rx_history` to match `post_rx_refill`?  Or alternatively if we want consistency of `rx` being at the end, we could change `post_rx_refill` below to be `post_refill_rx`?

File: lib/va_rx.rb [hunk 3]
```diff
@@ -2,7 +2,12 @@
...
-require 'va_rx/client'
+require 'va_rx/util'
 require 'va_rx/configuration'
 require 'va_rx/models/base'
+require 'va_rx/models/prescription'
 require 'va_rx/models/session'
+require 'va_rx/models/collection'
+require 'va_rx/api/prescriptions'
+require 'va_rx/api/sessions'
+require 'va_rx/client'
```
Comment (ayaleloehr) [inline lib/va_rx.rb]: would there be value in putting these alphabetically?  We should come up with a best practice around this.

File: lib/va_rx/configuration.rb [hunk 4]
```diff
@@ -12,7 +12,7 @@ def initialize(host:, app_token:, open_timeout: OPEN_TIMEOUT, read_timeout: READ
       @host = URI.parse(host)
       @open_timeout = open_timeout
       @read_timeout = read_timeout
-      assert_ssl
+      # assert_ssl
```
Comment (ayaleloehr) [inline lib/va_rx/configuration.rb]: Can you add a TODO here or say why it's commented out?

**Comment by ayaleloehr** [comment]
Mostly looks good with some questions back to you @saneshark . Please reassign to me after you either comment on those or address them, as appropriate.

File: lib/va_rx/api/prescriptions.rb (hunk 2 above)
Comment (saneshark) [inline lib/va_rx/api/prescriptions.rb]: agree, i will change this to get_rx_history for consistency with the others and we can revisit naming these all later if necessary

File: lib/va_rx.rb (hunk 3 above)
Comment (saneshark) [inline lib/va_rx.rb]: mostly they need to be ordered in such a way that any file that gets used by another is included first or else you end up with a Constant undefined error, but there is room for cleaning some of this stuff up. For example the module API in lib/api.rb would be a great place to list the api endpoints at least, and then we could just include that one file. But lets revisit this as part of refactor once I get specs up. I'll add a TODO for now.

**Comment by ayaleloehr** [comment]
LGTM

**Pull request merged**